Kinvey.Tests/BaseTestClass.cs

```diff
@@ -37,24 +37,23 @@ public static string AppKey
         {
             get
             {
-                return EnvironmentVariable.AppKey ?? "_kid_";
+                return TestSetup.app_key;

```

Comment:
maybe it's better to keep the `EnvironmentVariable.AppKey` so we can provide another value in runtime

Reply:
@heyzooi Ok. I will return environment variables back!